### PR TITLE
Automated cherry pick of #1408: Fix kube-proxy crashloopback for kind cluster

### PR DESCRIPTION
**Pull request by rene-dekker**
Cherry pick of #1408 on release-v1.15.

#1408: Fix kube-proxy crashloopback for kind cluster

### CODE REVIEW

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and leave the branch after I've merged it.

**Review by asincu — APPROVED** [review]

**Pull request merged**
